### PR TITLE
translations

### LINKED ISSUE #522: Translating FormEdit
Labels in FormEdit such as [this one](https://github.com/formio/react/blob/c966dd1d93d1ee5b5610496e2578ebc9ce153eb6/src/components/FormEdit.jsx#L63C77-L63C82) should be passed through i18next to allow for translations. The same goes for placeholder values.

 I am willing to make a PR. I just need confirmation that this change is desired.

### PR DESCRIPTION
**Pull request by PatrikTrefil**
fixes #522

### CODE REVIEW

**Comment by PatrikTrefil** [comment]
fixes #522

**Review by brendanbond — CHANGES_REQUESTED** [review]
Hey thanks again for the PR! The problem here though, is as I mentioned in the issue you opened: this library is designed to be a thin(ish) wrapper around the [formio.js](https://github.com/formio/formio.js) javascript form renderer. You can see that, for now, the core renderer [already loads i18next](https://github.com/formio/formio.js/blob/6627908117863247f0b87c0e746fde803b37578a/src/components/_classes/component/Component.js#L853) (as of the next major version, it will use a home-rolled solution for translation, but the point remains). 

So in other words, rather than load a new dependency in the react renderer, we merely need to access the underlying Component methods to translate.

**Comment by PatrikTrefil** [comment]
I do not really undestand how you think this should be implemented. At the level of the `FormEdit` component, which is the only component that requires the translations, we do not have any of the formiojs components available on which we could call the `t()` method. Since the `t()` method is not static, we need an instance to call it on. The instance of the `FormBuilder` class which is a subclass of the `Component` class is owned by the underlying `FormBuilder` react component, so we do not have access to it.

I think the issue is that the translation logic is written in the `Component` class. There should be a separate class with a singleton instance, which does the translation logic (similar design as the react-i18next library uses). This would require making changes in the formiojs library.

Maybe you have an idea how it should be implemented, but I just don't see it now. Please describe your solution further if you still think your solution is viable.

**Comment by PatrikTrefil** [comment]
Any updates on this please?

**Comment by brendanbond** [comment]
@PatrikTrefil hey thanks for  checking in, I'm actually working (broadly) on this issue now. I should have an alternate implementation hopefully very soon.
